dali/python/nvidia/dali/pipeline.py

```diff
@@ -62,11 +62,9 @@ class Pipeline(object):
 
     Parameters
     ----------
-    batch_size : int, optional, default = -1
-        Maximum batch size of the pipeline. Negative values for this parameter
-        are invalid - the default value may only be used with
-        serialized pipeline (the value stored in serialized pipeline
-        is used instead). In most cases, the actual batch size of the pipeline
+    batch_size : int
+        Maximum batch size of the pipeline. The value must be positivel
+        In most cases, the actual batch size of the pipeline
```

Comment:
Is the comment about serialized pipeline value not relevant anymore?

Also, there's a typo in word `positivel`